tests/accidental_tests.ts

```diff
@@ -161,11 +161,11 @@ function formatAccidentalSpaces(options: TestOptions): void {
     new StaveNote({
       keys: ['e##/5'],
       duration: '8d',
-    }).addModifier(0, new Accidental('##')),
+    }).addModifier(new Accidental('##'), 0),
```

Comment:
No change required, I would leave the explicit `0` with `Accidental`. Perhaps not in this case but indeed with Chords.